### ISSUE TITLE
Examples needed

### ISSUE THREAD

**Issue by akomakom** [jenkinsci/seed-plugin#21]
I apologize in advance for requesting more documentation in this manner (didn't find any other forum to do that, but hopefully this will be useful to others).

I've gone through the documentation and I'm just not getting it, and I could really use a complete example at all levels (seed, project seed, branch seed).

### Here is what I'm trying to do: 

- Automatically build branch-driven pipelines for a standard(ish) layout Subversion project.  For the moment let's disregard the fact that I already have a huge Job DSL framework for this and, for simplicity, let's just try for **seed/seed.groovy** per branch (I can always turn my code into a DSL library later).
- (Optional) Automatically tear down all of the generated jobs (about 100) when the branch is ... well, technically we don't delete branches so I don't know what I want here - some way of telling this plugin that we don't need to build this branch anymore, or switching to a subset of jobs for maintenance...   Editing seed.groovy may be fine here.

As much as possible, I'm going for **convention over configuration** here, so I'd like to leave all settings on defaults and adjust my layout accordingly.

### Problem

How do I set this up to automatically respond to branch creation/changes?

### Here is what I did so far:

Let's say that I'm setting up Seed to handle a big svn project at https://my.svn.server/svn/X (which of course has trunk, branches/*, tags/*, etc).  

1. I did not configure any Jenkins yaml settings in Global configuration.  From the documentation, it is unclear why I would need it if I'm following the Installation instructions that tell you to make a Seed job which takes all of the parameters explicitly.  

1. I did not configure any project-level seed settings.  From the Wiki, I didn't get a clear understanding of where that would go or why I would need it.  Does this go in the global yaml file?  Does it go in my project's codebase?

1. I did add a basic  https://my.svn.server/svn/X/trunk/seed/seed.groovy script that creates a test job.

1. Made a "seed" job that generated my "project seed" job (following Installation instructions).  I now have a folder with a [X]-seed job in it.  This job is parameterized and asks for the branch.  I understand that running it generates a branch-related job (this actually works and picks up my seed/seed.groovy).

But... I thought that the whole point was to have this happen **automatically** as branches come and go?  How do I tell the seed plugin, or the project seed job to watch https://my.svn.server/svn/X/trunk and https://my.svn.server/svn/X/branches/* ?   

I looked through the Branching Strategies page and it seems that it only talks about configuring the Jenkins job names and not the SCM branching conventions.

In summary, a complete SVN example would be most appreciated.

**Comment by dcoraboeuf** [comment]
Hi,

I'll try to answer very briefly (I'm not available a lot this week, I'll be back from holidays on next week...). If you want your branch folders and pipelines generated and started automatically, you have to send events from the SCM (SVN in your case) to the Jenkins instance. If this is fully documented for [GitHub](https://github.com/jenkinsci/seed-plugin/wiki/GitHub) or [BitBucket](https://github.com/jenkinsci/seed-plugin/wiki/BitBucket), this is unfortunately not the case for Subversion...

What I've done in the past for a client was to call the [HTTP end point](https://github.com/jenkinsci/seed-plugin/wiki/HTTP) from `post-commit` hooks installed in the Subversion server using Python. When I'm back, I'll try to provide those in the documentation.

As for your other points, you do not need the global configuration if you stick to the default settings and naming strategy (which is fine for me of course, but might be different for other projects, hence the need for a configuration...).

Best regards,
Damien.

**Comment by dcoraboeuf** [comment]
Hi,

Version 1.0-beta is now available and I think you'll find the [Quick Start](https://github.com/jenkinsci/seed-plugin/wiki/Quick-start) page very useful. Configuration of the plug-in is now much simpler.

Best regards,
Damien.
